packages/@dcl/inspector/src/components/Tree/Tree.tsx

```diff
@@ -31,147 +24,143 @@ type Props<T> = {
   canToggle?: (value: T) => boolean
   onSetParent: (value: T, parent: T) => void
   onRename: (value: T, label: string) => void
+  onAddComponent: (value: T, componentId: number) => void
```

Comment:
mm I'm worried that this component, that is supposed to be an abstract tree, it's growing too coupled with the composite inspector. For instance we could use this tree component to render the files in the Asset Explorer, but then we would need that tree to provide these methods that only make sense when `T` is an entity. I think I would rather add a `getContextMenu: (value: T) => MenuItem[]` function where each tree can provide custom actions for the context menu, and keep this component pure/abstract.